### PR TITLE
Update web tutorial to use new `dart` tool

### PR DESCRIPTION
**Pull request by shubhanshu02**
Fixes part of #2617. 
Update the commands in Web: Get Started Tutorial (`src/_tutorials/web/get-started.md`) to use the new `dart` tool

### CODE REVIEW

**Comment by shubhanshu02** [comment]
@kwalrath please review the changes.

File: src/_tutorials/web/get-started.md [hunk 1]
```diff
@@ -73,10 +71,8 @@ For a list of available IDEs, see the
 To create a web app from the command line, use these commands:
 
 ```terminal
-$ mkdir quickstart
+$ dart create -t web-simple quickstart
 $ cd quickstart
```
Comment (kwalrath) [inline src/_tutorials/web/get-started.md:75]: Let's move this down to the "Run the app" section (in the same code block as `webdev serve`).

**Review by kwalrath — APPROVED** [review]
Looks great! I'd just like to move one command.

File: src/_tutorials/web/get-started.md (hunk 1 above)
Comment (shubhanshu02) [reply src/_tutorials/web/get-started.md]: Sure

Comment (shubhanshu02) [reply src/_tutorials/web/get-started.md]: Sure, done!

**Comment by kwalrath** [comment]
Thank you, @shubhanshu02!

**Pull request merged**